From: Debian GnuPG Maintainers <pkg-gnupg-maint@lists.alioth.debian.org>
Date: Tue, 14 Apr 2015 10:02:31 -0400
Subject: avoid-beta-warning

avoid self-describing as a beta

Using autoreconf against the source as distributed in tarball form
invariably results in a package that thinks it's a "beta" package,
which produces the "THIS IS A DEVELOPMENT VERSION" warning string.

since we use dh_autoreconf, i need this patch to avoid producing
builds that announce themselves as DEVELOPMENT VERSIONs.

See discussion at:

 http://lists.gnupg.org/pipermail/gnupg-devel/2014-November/029065.html
---
 autogen.sh | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

Patch-Source: https://sources.debian.org/data/main/g/gnupg2/2.2.27-2/debian/patches/debian-packaging/avoid-beta-warning.patch

diff --git a/autogen.sh b/autogen.sh
index b238550..9b86d3f 100755
--- a/autogen.sh
+++ b/autogen.sh
@@ -229,7 +229,7 @@ if [ "$myhost" = "find-version" ]; then
     esac
 
     beta=no
-    if [ -e .git ]; then
+    if false; then
       ingit=yes
       tmp=$(git describe --match "${matchstr1}" --long 2>/dev/null)
       tmp=$(echo "$tmp" | sed s/^"$package"//)
@@ -245,8 +245,8 @@ if [ "$myhost" = "find-version" ]; then
       rvd=$((0x$(echo ${rev} | dd bs=1 count=4 2>/dev/null)))
     else
       ingit=no
-      beta=yes
-      tmp="-unknown"
+      beta=no
+      tmp=""
       rev="0000000"
       rvd="0"
     fi
